src/control/cmd/ddb/ddb_commands.go

```diff
@@ -0,0 +1,299 @@
+with the rm, load, commit_ilog, etc commands. The path <path> should be an absolute path to the
+pool shard. Part of the path is used to determine what the pool uuid is.`,
+		HelpGroup: "vos",
+		Flags: func(f *grumble.Flags) {
+			f.Bool("w", "write_mode", false, "Open the vos file in write mode.")
+		},
+		Args: func(a *grumble.Args) {
+			a.String("path", "Path to the vos file to open.")
+		},
+		Run: func(c *grumble.Context) error {
+			return ddbOpen(ctx, c.Args.String("path"), c.Flags.Bool("write_mode"))
+		},
+		Completer: openCompleter,
+	})
+	// Command: version
+	app.AddCommand(&grumble.Command{
+		Name:      "version",
+		Aliases:   nil,
+		Help:      "Print ddb version",
+		LongHelp:  "",
+		HelpGroup: "",
+		Run: func(c *grumble.Context) error {
+			return ddbVersion(ctx)
+		},
+		Completer: nil,
+	})
+	// Command: close
+	app.AddCommand(&grumble.Command{
+		Name:      "close",
+		Aliases:   nil,
+		Help:      "Close the currently opened vos pool shard",
+		LongHelp:  "",
+		HelpGroup: "vos",
+		Run: func(c *grumble.Context) error {
+			return ddbClose(ctx)
+		},
+		Completer: nil,
+	})
+	// Command: superblock_dump
+	app.AddCommand(&grumble.Command{
+		Name:      "superblock_dump",
+		Aliases:   nil,
+		Help:      "Dump the pool superblock information",
+		LongHelp:  "",
+		HelpGroup: "vos",
+		Run: func(c *grumble.Context) error {
+			return ddbSuperblockDump(ctx)
+		},
+		Completer: nil,
+	})
+	// Command: value_dump
+	app.AddCommand(&grumble.Command{
+		Name:      "value_dump",
+		Aliases:   nil,
+		Help:      "Dump a value",
+		LongHelp:  `Dump a value to the screen or file. The vos path should be a complete
+path, including the akey and if the value is an array value it should include
+the extent. If a path to a file was provided then the value will be written to
+the file, else it will be printed to the screen.`,
+		HelpGroup: "vos",
+		Args: func(a *grumble.Args) {
+			a.String("path", "VOS tree path to dump.")
+			a.String("dst", "File path to dump the value to.", grumble.Default(""))
+		},
+		Run: func(c *grumble.Context) error {
+			return ddbValueDump(ctx, c.Args.String("path"), c.Args.String("dst"))
+		},
+		Completer: nil,
+	})
+	// Command: rm
+	app.AddCommand(&grumble.Command{
+		Name:      "rm",
+		Aliases:   nil,
+		Help:      "Remove a branch of the VOS tree.",
+		LongHelp:  `Remove a branch of the VOS tree. The branch can be anything from a container
+and everything under it, to a single value.`,
+		HelpGroup: "vos",
+		Args: func(a *grumble.Args) {
+			a.String("path", "VOS tree path to remove.")
+		},
+		Run: func(c *grumble.Context) error {
+			return ddbRm(ctx, c.Args.String("path"))
+		},
+		Completer: nil,
+	})
+	// Command: value_load
+	app.AddCommand(&grumble.Command{
+		Name:      "value_load",
+		Aliases:   nil,
+		Help:      "Load a value to a vos path. ",
+		LongHelp:  `Load a value to a vos path. This can be used to update
```

Comment:
```suggestion
		Name:    "value_load",
		Aliases: nil,
		Help:    "Load a value to a vos path. ",
		LongHelp: `Load a value to a vos path. This can be used to update
```